### ISSUE TITLE
Not work in Cats XP

### ISSUE THREAD

**Issue by iAzamat** [floccusaddon/floccus#1614]
### Which version of floccus are you using?

5.1.2

### How many bookmarks do you have, roughly?

500

### Are you using other means to sync bookmarks in parallel to floccus?

Да

### Sync method

WebDAV

### Which browser are you using? In case you are using the phone App, specify the Android or iOS version and device please.

Catsxp version 4.5.2(125.0.6422.40) x64

### What kind of WebDAV server are you using? (if relevant)

My local webdav server in router keenetic

### Describe the Bug

If I install from the extension store, I get a white (empty) settings window. If I install manually, the sync button does not work

### Expected Behavior

The settings window should open and the synchronization button should work

### To Reproduce

Check on the latest version of Catsxp.
1. Install the extension from the store and get a white (empty) settings window.
2. Install the extension manually and find that the synchronization button does not work

P.S. The latest working version is 5.1.1, which I installed manually

### Debug log provided

- [ ] I have provided a debug log file

**Comment by marcelklehr** [comment]
this is a duplicate of #1613
